boards/arm/nucleo_l4r5zi/dts.fixup

```diff
@@ -0,0 +1,37 @@
+/* This file is a temporary workaround for mapping of the generated information
+ * to the current driver definitions.  This will be removed when the drivers
```

Comment:
What's going to remind someone to remove this workaround?  Is there an issue reported to track this?

Reply:
.fixup files are documented on http://docs.zephyrproject.org/devices/dts/device_tree.html
This is also visible at project level, but I haven't seen any GH issue to track this activity